index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section>
+      <header class="header-toolbar">
+        <a href="#" id="my-logo">
+          <h2 id="logo">My Logo</h2>
+        </a>
+        <nav id="vector-1">
+          <button>
+            <a href=""
+              ><img
+                src="./images/Vector button.png.jpeg"
+                alt="vector-button"
+                class="link"
+            /></a>
+          </button>
+        </nav>
```

Comment:
- [ ] [OPTIONAL] I would suggest not nesting an anchor tag `<a>` inside  `button` tag. You need  to inclose your `hamburger` button either in an `<a>` tag OR a`button `tag you don't need both. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. That is why it is best to not nest a button inside an anchor tag.

- [ ] [OPTIONAL] Also I would suggest you to remove default styling like border, outline from your hamburger to make your hamburger seamless e.g

```sh
button {
border: none;
outline: none;
background: transparent;
}
```

Your design: 

<img width="364" alt="Screenshot 2023-03-25 at 9 19 16 AM" src="https://user-images.githubusercontent.com/44798044/227697007-544934b3-76e2-44ee-b5bb-ff8e6c855821.png">

Figma Design:

<img width="202" alt="Screenshot 2023-03-25 at 9 19 08 AM" src="https://user-images.githubusercontent.com/44798044/227697020-91722f36-f809-412f-9170-24a873933967.png">